govc/vm/disk/attach.go

```diff
@@ -90,11 +90,7 @@ func (cmd *attach) Run(ctx context.Context, f *flag.FlagSet) error {
 	backing := disk.Backing.(*types.VirtualDiskFlatVer2BackingInfo)
 
 	if cmd.link {
-		if cmd.persist {
-			backing.DiskMode = string(types.VirtualDiskModeIndependent_persistent)
-		} else {
-			backing.DiskMode = string(types.VirtualDiskModeIndependent_persistent)
```

Comment:
Thanks for the PR @arbrix 
I think this should actually be VirtualDiskModeIndependent_nonpersistent here, rather than always setting to persistent.